src/main/resources/db/migration/V3__Create_project.sql

```diff
@@ -14,3 +14,8 @@ CREATE TABLE project (
 ALTER TABLE project ADD CONSTRAINT FK_PROJECT_ON_CUSTOMER FOREIGN KEY (customer_id) REFERENCES customer (id);
 
 ALTER TABLE project ADD CONSTRAINT FK_PROJECT_ON_EMPLOYEE FOREIGN KEY (employee_id) REFERENCES employee (id);
+
+INSERT INTO project(customer_id,employee_id,name,description,starting_date,created,updated)
+VALUES
+    (1,1,'test','testing',CURRENT_DATE,CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+    (1,1,'test2','testing delete',CURRENT_DATE,CURRENT_TIMESTAMP, CURRENT_TIMESTAMP)
```

Comment:
I tried putting there into file `afterMigrate.sql` in `test/resouces/db.migration` but for whatever reason it did not work. Tried to follow this documentation:
https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto.data-initialization.migration-tool.flyway

> You can also use Flyway to provide data for specific scenarios. For example, you can place test-specific migrations in `src/test/resources` and they are run only when your application starts for testing.

But apparently some other changes are needed too, so still having the test data in the migration files

Reply:
Yeah that's fine. It might need some more configurations in the integrationtest base class to let it know migrations are available. Let's do another issue about that. But you are correct that these should absolutely not be under main/resources and started accumulating here out of my laziness.